### ISSUE TITLE
MS Teams: [Chat - Files tab]: NVDA users are not able to navigate to "Share" controls in browse mode under Chat> Files tab

### ISSUE THREAD

**Issue by HarishTerala** [nvaccess/nvda#10505]
### Steps to reproduce:
1. Open MS Teams app.
2. Open the one to one chat of a new user (not yet started any chat yet).
3. Navigate to "Files" tab and activate it.
4. Try to navigate to "Share" button in browse mode and activate it.

### Actual behavior:
NVDA users are not able to navigate to "Share" controls, when focus reaches around it NVDA reads as "File action toolbar".

### Expected behavior:
NVDA users should be able to navigate and activate "Share" controls in browse mode.

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
2019.2.1

#### Windows version:
Win10 1903

#### Does the issue still occur after restarting your PC?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No

**Comment by ehollig** [comment]
I am able to activate the share control in focus mode. Once activating the files tab, either in browse or focus mode, tab to the share control and press enter. NVDA reads, "Share, share files  1 of 1". It then brings up a menu allowing you to upload from OneDrive, and pressing down arrow will allow you to upload from your computer.

**Comment by HarishTerala** [comment]
@ehollig - Issue is in the browse mode, when we navigate using down arrow key first time it read as "File actions toolbar" again pressing down arrow it reads as "list". Here NVDA is not reading the "Share" control information and pressing Enter/Space key will not perform any action.

**Comment by Adriani90** [comment]
This might be due to the fact that the share button is not a button but a list item, and I think that item has a role="application" on it without role="document". Without role="document", on top of role="application" NVDA will not be able to detect it in browse mode.

This seems an application design issue, but I assume the same principal problem like in #10529.

**Comment by Adriani90** [comment]
@HarishTerala are you still having this issue with NVDA 2024.4 and the new MS Teams, or while using the browsers last version?

**Comment by Adriani90** [comment]
This is not reproducible in MS Teams Mini with NVDA last alpha. Tested in Teams 24335.208.3315.1951  / Clientversion 49/24120100212.
Closing as works for me.
